python/cuml/dask/ensemble/base.py

```diff
@@ -264,6 +287,17 @@ def _func_fit(model, input_data, convert_dtype):
     return model.fit(X, y, convert_dtype)
 
 
+def _func_predict(model, input_data, **kwargs):

```

Comment:
maybe `_func_predict_partial` to clarify? Also a comment explaining how it works